tests/integration/tests/autocomplete.js

```diff
@@ -42,6 +41,24 @@ test('search and clear', async () => {
   expect(searchValueAfterClear).toEqual('')
 })
 
+test('search has lang in query', async () => {
+  const langPage = await browser.newPage()
+  await langPage.setDefaultTimeout(2000)
```

Comment:
is it stil required ?

Reply:
Not required. But I would keep it, as it could help with debugging in the future....

Reply:
I will add a comment about it.